cmd/utils/flags.go

```diff
@@ -592,6 +592,39 @@ var (
 		Name:  "anchoring",
 		Usage: "Enable anchoring for service chain",
 	}
+	// KAS
+	ServiceChainKASAnchorFlag = cli.BoolFlag{
```

Comment:
As the namespace begins with `kas`, how about changing the prefix of variables from `ServiceChainKAS` to `KASServiceChain`?

Reply:
Reflected